### PR TITLE
Fix ETW init failure

### PR DESCRIPTION
**Pull request by heyams**
this is a regression caused by moving `com.microsoft.applicationinsights.agent.bootstrap.diagnostics.etw` to `com.microsoft.applicationinsights.agent.internal.diagnostics.etw` (#3190)

### CODE REVIEW

File: .github/workflows/build-common.yml [hunk 1]
```diff
@@ -152,7 +152,7 @@ jobs:
         run: |
           modules=$(ls -d smoke-tests/apps/* | sed 's/\/$//' | sed 's/\//:/g' | sed 's/^/:/')
           inner_json=$(echo $modules | xargs echo | sed 's/ /","/g')
-          echo "::set-output name=matrix::{\"module\":[\"$inner_json\"]}"
+          echo "{matrix}={\"module\":[\"$inner_json\"]}" >> $GITHUB_OUTPUT
```
Comment (heyams) [inline .github/workflows/build-common.yml:155]: https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/

File: etw/java/src/main/jni/com/microsoft/applicationinsights/agent/internal/diagnostics/etw/EtwProvider.java [hunk 2]
```diff
@@ -25,7 +25,7 @@ public EtwProvider(String sdkVersion) {
         LOGGER.debug("EtwProvider initialized. Lib path={}", dllPath.getAbsolutePath());
       } catch (Throwable t) {
         try {
-          LOGGER.error("Error initializing EtwProvider", t);
+          LOGGER.debug("Error initializing EtwProvider", t);
```
Comment (heyams) [inline etw/java/src/main/jni/com/microsoft/applicationinsights/agent/internal/diagnostics/etw/EtwProvider.java:28]: we should log anything under debug logging level for ETW related errors.

**Review by trask — APPROVED** [review]

File: etw/java/src/main/java/com/microsoft/applicationinsights/agent/internal/diagnostics/etw/DllFileUtils.java [hunk 3]
```diff
@@ -78,9 +83,9 @@ public static void extractToLocalFolder(File dllOnDisk, String libraryToLoad) th
         while ((bytesRead = in.read(buffer)) != -1) { // while not EOF
           out.write(buffer, 0, bytesRead);
         }
+        LOGGER.debug("Successfully extracted '{}' to local folder", libraryToLoad);
       }
     }
-    LOGGER.debug("Successfully extracted '{}' to local folder", libraryToLoad);
```
[the remark this answers is not in the document]
Comment (heyams) [reply etw/java/src/main/java/com/microsoft/applicationinsights/agent/internal/diagnostics/etw/DllFileUtils.java:83]: i can revert it back

**Review by jvano — APPROVED** [review]
Looks good to me. :shipit:

**Comment by heyams** [comment]
> Looks good to me. :shipit:

Thank you @jvano

**Pull request merged**